--- amr-release-proxy.txt	2014-04-18 16:24:38.000000000 -0400
+++ amr-release-proxy.txt.FIXED	2014-04-18 16:24:55.000000000 -0400
@@ -11394,7 +11394,7 @@
           :op2 (c2 / country
                  :name (n2 / name :op1 "Iran"))
           :op3 (c3 / country
-                 :name (n / name :op1 "Pakistan"))))
+                 :name (n4 / name :op1 "Pakistan"))))
 
 # ::id PROXY_AFP_ENG_20020518_0107.7 ::date 2013-06-08T16:34:58 ::snt-type body ::annotator LDC-AMR-01 ::preferred
 # ::snt Iran and Pakistan plan to increase private sector activity with Afghanistan.
@@ -54889,7 +54889,7 @@
       :op2 (p / project-02
             :ARG1 (o / operate-01
                   :ARG1 p2
-                  :time (a / after
+                  :time (a2 / after
                         :op1 (n2 / now)
                         :quant (t / temporal-quantity :quant 7
                               :unit (y / year))))))
@@ -74986,7 +74986,7 @@
                         :ARG1 c3
                         :mod (c4 / cyber)
                         :quant (s / series))))
-      :time (b / before
+      :time (b3 / before
             :op1 (n / now)
             :quant (t2 / temporal-quantity :quant 3
                   :unit (w3 / week))))
@@ -75007,7 +75007,7 @@
                         :op2 (w3 / website
                               :mod (p / private)
                               :quant (s / some)))))
-      :time (b / before
+      :time (b3 / before
             :op1 (n2 / now)
             :quant (t / temporal-quantity :quant 3
                   :unit (w2 / week)))
@@ -161981,7 +161981,7 @@
             :ARG1 (p2 / purchase-01 :polarity -
                   :ARG0 (c / country :name (n2 / name :op1 "Venezuela"))
                   :ARG1 (a2 / arm)
-                  :time (a / after
+                  :time (a4 / after
                         :op1 (n5 / now)
                         :quant (t / temporal-quantity :quant 25
                               :unit (y / year)))))
